boards/nxp/mcx_n9xx_evk/doc/index.rst

```diff
@@ -0,0 +1,361 @@
++-----------+------------+-------------------------------------+
+| USBHS     | on-chip    | USB High-Speed device               |
++-----------+------------+-------------------------------------+
+| USBFS     | on-chip    | USB Full-Speed device               |
++-----------+------------+-------------------------------------+
+| LPCMP     | on-chip    | sensor(comparator)                  |
++-----------+------------+-------------------------------------+
+| FLEXCAN   | on-chip    | CAN                                 |
++-----------+------------+-------------------------------------+
+| LPTMR     | on-chip    | counter                             |
++-----------+------------+-------------------------------------+
+| FLEXIO    | on-chip    | flexio                              |
++-----------+------------+-------------------------------------+
+| SAI       | on-chip    | i2s                                 |
++-----------+------------+-------------------------------------+
+| DISPLAY   | on-chip    | flexio; MIPI-DBI. Tested with       |
+|           |            | :ref:`lcd_par_s035`                 |
++-----------+------------+-------------------------------------+
+| MRT       | on-chip    | counter                             |
++-----------+------------+-------------------------------------+
+
+Dual Core samples
+*****************
+
++-----------+-------------------+----------------------+
+| Core      | Boot Address      | Comment              |
++===========+===================+======================+
+| CPU0      | 0x10000000[1856K] | primary core flash   |
++-----------+-------------------+----------------------+
+| CPU1      | 0x101d0000[192K]  | secondary core flash |
++-----------+-------------------+----------------------+
+
++----------+------------------+-----------------------+
+| Memory   | Address[Size]    | Comment               |
++==========+==================+=======================+
+| srama    | 0x20000000[320k] | CPU0 ram              |
++----------+------------------+-----------------------+
+| sramg    | 0x20050000[64k]  | CPU1 ram              |
++----------+------------------+-----------------------+
+| sramh    | 0x20060000[32k]  | Shared memory         |
++----------+------------------+-----------------------+
+
+Targets available
+==================
+
+The default configuration file
+:zephyr_file:`boards/nxp/mcx_n9xx_evk/mcx_n9xx_evk_mcxn947_cpu0_defconfig`
+only enables the first core. CPU0 is the only target that can run standalone.
+
+CPU1 does not work without CPU0 enabling it.
+
+To enable CPU1, create System Build application project and enable the
+second core with config :kconfig:option:`CONFIG_SECOND_CORE_MCUX`.
+
+Please have a look at some already enabled samples:
+
+- :zephyr_file:`samples/subsys/ipc/ipc_service/static_vrings`
+- :zephyr_file:`samples/subsys/ipc/openamp`
+- :zephyr_file:`samples/drivers/mbox`
+- :zephyr_file:`samples/drivers/mbox_data`
```

Comment:
Use `:zephyr:code-sample:` to reference code samples by name

Reply:
Good point, updated